fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tags/v1/CommonItemTags.java

```diff
@@ -0,0 +1,69 @@
+package net.fabricmc.fabric.api.tags.v1;
+
+import net.minecraft.item.Item;
+import net.minecraft.tag.TagKey;
+
+import net.fabricmc.fabric.impl.v1.TagRegistration;
+
+public class CommonItemTags {//todo include fabric tools in common tools
+	public static final TagKey<Item> PICKAXES = register("pickaxes");
+	public static final TagKey<Item> SHOVELS = register("shovels");
+	public static final TagKey<Item> HOES = register("hoes");
+	public static final TagKey<Item> AXES = register("axes");
+	public static final TagKey<Item> SHEARS = register("shears");
+	/**
+	 * For throwable weapons, like Minecraft tridents
+	 */
+	public static final TagKey<Item> SPEARS = register("spears");
+	public static final TagKey<Item> SWORDS = register("swords");
+	public static final TagKey<Item> BOWS = register("bows");
+	/**
+	 * @deprecated Use {@link CommonItemTags#PICKAXES}
```

Comment:
Imo these should not be in the API at all.

Reply:
I'm going to disagree - FAPI had tags for swords, and it is useful to know when a mod adds a sword that is not a SwordItem.

I'm open to discussion on bows, but we include tags for all the other tools anyways, there is no reason to leave a hole.

Reply:
I think what Technici4n means are the deprecated tags, FabricToolTags was removed in favor of `mineable/tool` block tags.

Reply:
No, they weren't - mineable tags do not replicate nor replace the function of tool tags at all.
Tool tags - "this is a pickaxe"
Mineable tags - "this block prefers to be broken by a pickaxe"

Reply:
Sure, but it still doesn't change the fact that they're removed from FAPI code. If you tell people to use the `c` tags, people probably will use that, making the `fabric` ones useless. If you still care about mods that still uses fabric tool tags, put it on the json/datagen.

Reply:
I've moved them to exist within the datagen only.